runner/batch.go

```diff
@@ -0,0 +1,49 @@
+package runner
+
+import (
+	"errors"
+	"os"
+	"strings"
+)
+
+type BatchJob struct {
+	JobId              string
+	Queue              string
+	ComputeEnvironment string
+	DependencyIds      []string
+}
```

Comment:
without a good understanding of AWS Batch, it's hard to get oriented on what this struct represents. Comments on this struct + field values would make it easier!